maxim/apis/maxim_apis.py

```
@@ -48,22 +64,35 @@ def __init__(self):
         retries = Retry(
-            connect=5,
+            total=5,
+            connect=3,
             read=3,
             redirect=1,
             status=3,
-            backoff_factor=0.4,
+            backoff_factor=0.5,
             status_forcelist=frozenset({413, 429, 500, 502, 503, 504}),
+            allowed_methods=frozenset(
+                {"HEAD", "GET", "PUT", "DELETE", "OPTIONS", "TRACE", "POST"}
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)